src/Components/Sidebar/SidebarOption.tsx

```diff
@@ -0,0 +1,18 @@
+import { boolean } from "yargs";
+import {SideBarOption} from "../../Types/types"
+type SideBarOptionType = {
+  option: SideBarOption;
+  isActive: boolean;
+  handlePageClick: (selectedPage: string ) => void
+}
+export default function SidebarOption({ option, isActive, handlePageClick }: SideBarOptionType) {
+  const classList = "sidebar-option" + (isActive ? " active" : "");
+  return (
+    <div className={classList} onClick={() => handlePageClick(option.name)}>
+      <div className="sidebar-icon">{option.icon != null && option.icon}</div>
```

Comment:
No need to use this condition.
icon is of type JSX.Element it will never be undefined | null.